.github/workflows/test.yaml

```diff
@@ -154,7 +153,34 @@ jobs:
           nslookup github-actions.qhub.dev
       - name: Curl jupyterhub login page
         run: |
-          curl -k https://github-actions.qhub.dev/hub/home -i
+          curl -k --retry 5 --retry-connrefused https://github-actions.qhub.dev/hub/login
```

Comment:
`curl -k https://github-actions.qhub.dev/hub/home -i` should now work. Retry is no longer needed